kuma/static/js/highlight.js

```diff
@@ -1,16 +1,19 @@
+// The React version of this code is in the highlightSections() function
+// of kuma/javascript/src/article.jsx
 (function($) {
     'use strict';
 
     // find all targets and wrap appropriately
     function highlight(targets) {
         var $targets = $(targets);
         $targets.each(function() {
-            $(this).addClass('highlight-spanned').wrapInner('<span class="highlight-span"></span>');
+            $(this)
+                .addClass('highlight-spanned')
+                .wrapInner('<span class="highlight-span"></span>');
```

Comment:
prettier rewrapped this one line when I added the comment above. It seemed harmless so I left the change in. Let me know if you'd like me to avoid reformatting these older JS files.